packages/plugin-php/index.js

```diff
@@ -5,8 +5,18 @@ export default {
   name: 'PHP',
 
   resolve(path, [target], meta, regExp) {
+    if (target.startsWith('Illuminate')) {
+      return liveResolverQuery({
+        type: 'ping',
+        target: `https://laravel.com/api/8.x/${target.replace(
+          /\\/g,
+          '/',
+        )}.html`,
```

Comment:
Should we move this out to a helper function or maybe move it down lower in the api? It's being done over here too https://github.com/OctoLinker/OctoLinker/blob/4317c15952d159ef6d503b7f8b887666b86fc8c7/packages/plugin-dotnet-project/index.js#L8-L9

Reply:
This was just a quick try to test if this is working. I'm currently consider to add this logic to the OctoLinker API so we can maintain this more easily. Also I would like to avoid such a mapping list to make it work for all api docs out of the box which might be tricky or even impossible.